### ISSUE TITLE
[Tools QOL] Reset property to default value button

### ISSUE THREAD

**Issue by KayaDLX** [Facepunch/sbox-issues#3032]
### For?

Hammer

### What can't you do?

UE5 does this cool thing where it just shows a button next to the property to quickly restore default value, this is faster than right clicking and finding the restore default value button. This is just a small QOL change that could be cool to have.

### How would you like it to work?

![image](https://user-images.githubusercontent.com/67245359/225783749-a0e904cc-3578-4d09-a4fa-1c845ee0a3e3.png)
Pop a nice button next to properties to reset them with one click.

**Comment by robotboy655** [comment]
Duplicate of https://github.com/sboxgame/issues/issues/2915

**Comment by DoctorGurke** [comment]
Not duplicate, he's asking for a dedicated button. Like in hammer and other tools that already have the reset to default function

**Comment by KayaDLX** [comment]
^

**Comment by KayaDLX** [comment]
Yeah understandable, even if those eventually get remade, it's a good QOL idea to keep in mind when that comes 🙏

**Comment by DoctorGurke** [comment]
Good first step would be to combine these two issues and add the buttons to the property sheet widget then ;)
I saw garry mention he wants to rework them somewhat anyways
